src/prommis/uky/uky_flowsheet.py

```diff
@@ -1698,6 +1698,7 @@ def initialize_system(m):
     ]
 
     initializer_sx = SolventExtractionInitializer()
+    # TODO: Initialize m.fs.solex_rougher_strip with SolventExtractionInitializer
```

Comment:
What is the effort level of doing this as part of this PR?

Reply:
I'm unsure... this may be coincidentally resolved as I try to improve scaling/initialization. Otherwise, it should be resolve in a different PR

Reply:
After playing around with things, I think this should be addressed in a separate PR